docs/verifiable-document/document-store.md

```diff
@@ -6,19 +6,48 @@ sidebar_label: Deploying Document Store
 
 The document store is a smart contract on the Ethereum network that records the issuance and revocation status of OA documents. In this guide, we will deploy a document store smart contract on the Ethereum `ropsten` network, which is a test network that does not require actual [ethers](/docs/appendix/glossary#ether) for transactions.
 
-## Prerequisites
-
-- Google Chrome web browser
-- Metamask setup, refer to the [Test Account Setup](/docs/appendix/ropsten-setup) section for the setup guide
-
 ## Deployment Methods
 
 Currently, we provide two ways of deploying the document store smart contract onto Ethereum:
 
-- [TradeTrust Admin Portal](https://admin.tradetrust.io/), a web app
+- [TradeTrust Admin Portal](https://admin.tradetrust.io/) or [Opencerts Admin Portal](https://admin.opencerts.io/) , a web app
```

Comment:
formatting:
- i think Opencerts is stylized as OpenCerts
- additional whitespace after the link and before the comma

Reply:
done